From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sun, 13 Jan 2019 08:42:32 +0900
Subject: man: fix reference

Fixes #11396.

(cherry picked from commit 227bcd91b42cc9291bc6539bb6127af74fe5e466)
(cherry picked from commit 2916f4eb8f57888b34400e500550ab51d1b6259f)
---
 man/sd_event_wait.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/sd_event_wait.xml b/man/sd_event_wait.xml
index 8849962..f01d18e 100644
--- a/man/sd_event_wait.xml
+++ b/man/sd_event_wait.xml
@@ -157,7 +157,7 @@
         <listitem><para>An event source is currently being prepared,
         i.e. the preparation handler is currently being executed, as
         set with
-        <citerefentry><refentrytitle>sd_event_set_prepare</refentrytitle><manvolnum>3</manvolnum></citerefentry>. This
+        <citerefentry><refentrytitle>sd_event_source_set_prepare</refentrytitle><manvolnum>3</manvolnum></citerefentry>. This
         state is only seen in the event source preparation handler
         that is invoked from the
         <function>sd_event_prepare()</function> call and is
